src/plugins/discover/public/application/main/components/pattern_analysis/pattern_analysis_table.tsx

```diff
@@ -63,8 +64,9 @@ export const PatternAnalysisTable = (props: PatternAnalysisTableProps) => {
   return (
     <aiopsService.PatternAnalysisComponent
       props={patternAnalysisComponentProps}
-      deps={services}
-      embeddingOrigin="discover"
+      appContextValue={
+        { embeddingOrigin: 'discover', ...services } as unknown as AiopsAppContextValue
```

Comment:
This casting to `unknown` was previously done inside the component. I was able to remove one layer of adjusting dependencies but it made this necessary on the consuming code. We plan to get rid of the `unknown` in a follow up.

Reply:
Will this be done as an immediate followup? If not, could you leave a comment above the line indicating that it will soon be updated with a link to the followup issue or this PR?